File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.js
function solution(a, b) {
    // 두 정수를 문자열로 변환하여 연결하고, 다시 정수로 변환
    const op1 = parseInt(a.toString() + b.toString());
    // 주어진 공식에 따라 계산
    const op2 = 2 * a * b;

    // 두 연산의 결과를 비교하여 더 큰 값을 반환
    // 같은 경우에는 op1을 반환
    return op1 >= op2 ? op1 : op2;
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        // a와 b를 문자열로 변환하여 붙인 후 정수로 변환하여 비교
        int num1 = int.Parse(a.ToString() + b.ToString());
        int num2 = 2 * a * b;

        // 두 값을 비교하여 큰 값을 반환
        return Math.Max(num1, num2);
    }
}